Remove double-colon from clean target which causes breakage in reverse-depends.

--- petsc-3.1.dfsg/conf/rules~	2008-12-19 23:48:45.000000000 -0500
+++ petsc-3.1.dfsg/conf/rules	2009-06-16 14:21:12.000000000 -0400
@@ -117,7 +117,7 @@
 	@${CP} -f *.mod ${PETSC_DIR}/${PETSC_ARCH}/include
 
 # Removes garbage files
-clean::
+clean:
 	@-${RM} *.o *.lo *~ ex[0-9] ex[0-9][0-9] ex[0-9]f ex[0-9][0-9]f \
 	       ${CLEANFILES} ${TESTS} ex[0-9]f90 ex[0-9][0-9]f90 ex[0-9]c ex[0-9][0-9]c ex[0-9]d ex5d \
 	       PI* *.ln l.outa* mputil.mp_* core core.* *.tmp *.map gmon.out \
